src/pages/Filing/PrivacyNotice.tsx

```diff
@@ -10,7 +10,7 @@ function PrivacyNotice(): ReactElement {
       <Layout.Wrapper>
         <Layout.Content flushBottom>
           <CrumbTrail>
-            <Link href='/'>Platform home</Link>
+            <Link href='/landing'>Platform home</Link>
```

Comment:
We don't want this to be `/landing`, because otherwise if a user isn't logged in like when they first come to the home page, this link won't work. We could do some conditional routing if we end up putting this on the authenticated home page down the line like Natalia mentioned today after mvp, routing them back depending on where they're coming from, but for now, better to keep it just `/`.